dc.sh

```diff
@@ -13,7 +13,7 @@ get_tt_env() {
 }
 
 dc() {
-	env $(get_tt_env) docker-compose -f $(dc_file) $*
+	env $(get_tt_env) docker-compose -p $(get_workspace) -f $(dc_file) $*
```

Comment:
this will fix `tt workspace add foo /tmp/bar` making containers under the name `bar` rather than `foo`?

this is a big one! enforcing sanitised directory names sucks, but we can enforce alias names! (eventually)

Reply:
Yes, the container names end up based on the workspace name (which is nice). I think your example is backward, though.

It does, however, mean you'll need to destroy and recreate workspace containers. Might need to do that before upgrading, hmm.

Reply:
yeah meh breaking change is fine, folder and ws is the same name usually anyway

I worded my comment weird, (it's fixing that backwards behavior) but yeah that's what I meant